server/graphql/schema/mutations/createUser.ts

```diff
@@ -31,6 +33,8 @@ export const createUser = {
   async resolve(parent: object, args: object) {
     const user = await User.create({ ...args, isAdmin: false });
 
+    sendEmail(user.email);
```

Comment:
this should be `await` when `sendEmail` becomes async func

Reply:
okok